File: backend/servicio/models/postulacion.js
const Sequelize = require('sequelize');
module.exports = function(sequelize, DataTypes) {
  return sequelize.define('postulacion', {
    id: {
      autoIncrement: true,
      type: DataTypes.INTEGER.UNSIGNED,
      allowNull: false,
      primaryKey: true
    },
    puesto_id: {
      type: DataTypes.INTEGER.UNSIGNED,
      allowNull: true
    },
    pasante: {
      type: DataTypes.INTEGER,
      allowNull: true
    },
    estado: {
      type: DataTypes.SMALLINT,
      allowNull: true
    },
    fecha: {
      type: DataTypes.DATEONLY,
      allowNull: true
    }
  }, {
    sequelize,
    tableName: 'postulacion',
    timestamps: false,
    indexes: [
      {
        name: "PRIMARY",
        unique: true,
        using: "BTREE",
        fields: [
          { name: "id" },
        ]
      },
    ]
  });
};

File: frontend/passantic/lib/models/postulacion.dart
import 'package:flutter/material.dart';
import 'package:passantic/models/pasante.dart';

class Postulacion {
  final int id, estado,puestoid;
  final String fecha;
  final Pasante pasante;
  final String cargo;
  Postulacion(this.id, this.cargo, this.pasante, this.estado, this.fecha,this.puestoid);

}
